lib/internal/test_runner/reporter/spec.js

```diff
@@ -35,114 +35,116 @@ const symbols = {
   'arrow:right': '\u25B6 ',
   'hyphen:minus': '\uFE63 ',
 };
-class SpecReporter extends Transform {
```

Comment:
@MoLow @cjihrig Do we still want/need this? 


https://github.com/nodejs/node/blob/c462147a6314b82ec461029d9789bb8c738aab44/lib/internal/test_runner/utils.js#L142-L144

The docs do not show passing a constructor as an option...

Reply:
yes, for custom reporters I guess

Reply:
Then I guess we might want to add an example for that to the docs?